packages/insomnia-testing/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "include": [
+    "webpack.config.js",
+    "jest.config.js",
+    ".bablerc.js",
```

Comment:
good call.  it was there at some stage, but since we are just using the typescript compiler now via ts-loader it isn't needed anymore.  we were _just barely_ using it before, anyway.

Reply:
resolved in https://github.com/dimitropoulos/insomnia/pull/6/commits/da6d731cdf4a1335a6cb9a970e26ac1a9e9761d0